### PR TITLE
SfB online retirement and transition content

### PR DESCRIPTION
**Pull request by tophpalmer**

### CODE REVIEW

**Comment by tophpalmer** [comment]
@lindalu-MSFT , this is a big edit, could you please review and provide feedback. I'm MSFT internal and working the SfB retirement.

I'm sorry if I broke the OPS build, I'm not sure what tags/table is the standard header in this repo.

**Comment by lindalu-MSFT** [comment]
@tophpalmer Hi, sorry I overlooked this. I can help you fix this stuff, but it will faster if I just show you? I see three issues:
1. Check other topics in this repo for a sample of what our metadata tags at the top look like for directions.
2. Remove /en-us/ from all the URLS
3. The msdn links are all redirected. Just follow the link and copy and paste the new URL (remove en-us first.)

Let me know if you want to have a quick chat over Teams and I'll show you.

File: Skype/AppSDK/SkypeAppSDK.md [hunk 1]
```diff
@@ -1,16 +1,12 @@
-
+---
+title: Skype for Business App SDK
+description: Introduction to the Skype for Business App SDK
+---
```
Comment (lindalu-MSFT) [inline Skype/AppSDK/SkypeAppSDK.md:4]:
```suggestion
ms.date: 02/10/2021
---
```

File: Skype/AppSDK/SkypeAppSDK.md [hunk 2]
```diff
@@ -26,3 +22,35 @@ The Skype App SDK documentation consists of the following sections:
...
+
+Azure Communication Services is a cloud-based communications service that lets you add voice, video, chat, and telephony to your apps. ACS applications have the ability to join Teams meetings (as a guest), and we encourage implementers of the SfB App SDK to transition to Azure Communication Services. The ability to join Teams meetings is in active development and restricted to a private preview at this time. 
+
+
+### Azure SDKs that join Teams meetings
+
+Azure Communication Services has multiple SDKs that can join Teams meetings listed below. The Meeting UI SDK is available for iOS and Android and is the closest equivalent to SfB Mobile, requiring the least amount of code and providing an all-in-one Teams styled interface. For most SfB App SDK customers, this is the best option. 
+
+|SDK| Implementation Complexity|	Customization Ability|	Calling| Chat| [Join Teams Meetings](https://docs.microsoft.com/azure/communication-services/concepts/voice-video-calling/teams-interop)
```
Comment (lindalu-MSFT) [inline Skype/AppSDK/SkypeAppSDK.md:38]:
```suggestion
| SDK | Implementation Complexity | Customization Ability| Calling | Chat | [Join Teams Meetings] |(https://docs.microsoft.com/azure/communication-services/concepts/voice-video-calling/teams-interop)
```

**Review by lindalu-MSFT — APPROVED** [review]

**Pull request merged**